CMakeLists.txt

```diff
@@ -864,7 +864,9 @@ SUBDIRS(src)
 IF(NOT(QMC_BUILD_LEVEL GREATER 4))
   SUBDIRS(tests)
   SUBDIRS(examples)
-  SUBDIRS(nexus/examples)
+  IF(IS_DIRECTORY nexus)
```

Comment:
Why is this check needed?  Is the nexus directory not present sometimes?

Reply:
Yes. I do not always checkout it.